daemon/src/payout_curve.rs

```diff
@@ -92,6 +95,8 @@ fn calculate_payout_parameters(
         None,
     )?;
 
+    let funding_fee = funding_params.calculate_fee()?;
```

Comment:
Why do we need to pass `FundingParams` all the way through if we end up only calling a single function here? Seems to be cleaner to do this outside and just pass the result in?

Reply:
interesting. I wanted strong typing here, but indeed another newtype that's the result of this method would probably be better to be passed in.